docs/user/file-size.md

```diff
@@ -75,3 +75,9 @@ with open("out.pdf", "wb") as f:
 
 Using this method, we have seen a reduction by 70% (from 11.8 MB to 3.5 MB)
 with a real PDF.
+
+## Removing Sources
+
+Often times when deleting pages, the source will persist. This will reduce the page count, while marginally reducing the file size. Make sure to remove all refrences to objects inorder to remove the sources. 
```

Comment:
```suggestion
When a page is deleting (removed from page list), the source will persist(the data may be still be used somewhere else). This will reduce the page count, while not reducing the file size. 
To not include the code, the pages should not be added using `PdfWriter.append()` selecting the good pages only
```